Server/controllers/authController.js

```
@@ -112,47 +116,44 @@
+    const user = await req.db.getUserByEmail(email);
+
+    // Compare password
+    const match = await user.comparePassword(password);
+    if (match !== true) {
+      next(new Error(errorMessages.AUTH_INCORRECT_PASSWORD));
+      return;
     }
+
+    // Remove password from user object.  Should this be abstracted to DB layer?
+    const userWithoutPassword = { ...user._doc };
+    delete userWithoutPassword.password;
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)